commitlint.config.ts

```diff
@@ -333,7 +333,9 @@ module.exports = {
         'proper-issue-refs': [RuleStatus.Error, 'always'],
         'too-many-spaces': [RuleStatus.Error, 'always'],
         'commit-hash-alone': [RuleStatus.Error, 'always'],
+        'proper-revert-message': [RuleStatus.Error, 'always'],
     },
+    defaultIgnores: false,
```

Comment:
@realmarv as it's not very clear why this is needed, let's add a comment that explains it; after that, I'll merge this PR

Reply:
Done

Reply:
if done, click on "Resolve conversation" please